setup.py

```diff
@@ -2,6 +2,7 @@
 from setuptools.command.test import test as TestCommand
 import sys
 
+from ducktape.utils.util import ducktape_version
```

Comment:
No I'm not sure. Switched the the mechanism used in requests where the setup.py file actually greps for the version